src/token_contract/src/test/mint_to_commitment.nr

```diff
@@ -1,17 +1,18 @@
 use crate::{test::utils, Token};
 use aztec::oracle::random::random;
+use uint_note::uint_note::PartialUintNote;
 
 #[test]
 unconstrained fn mint_to_commitment_success() {
     // Setup without account contracts. We are not using authwits here, so dummy accounts are enough
     let (env, token_contract_address, _, recipient, minter) = utils::setup_with_minter(false);
 
+    env.impersonate(minter);
     let commitment = Token::at(token_contract_address)
-        .initialize_transfer_commitment(recipient, recipient)
+        .initialize_transfer_commitment(minter, recipient)
```

Comment:
this is only for msg encryption, am i right? we could have just left `recipient` here?

Reply:
hmmm this mixed behaviour is weird, let's say completer needs to be a smart-contract (w/o encrypting keys), then how's this supposed to work?
imo, completer should always be msg sender